### DISCUSSION TITLE
Multi-sample integration done on the "peaks" or "ATAC" assay?

### DISCUSSION THREAD

**Discussion by HaglundA** [stuart-lab/signac#636] [Q&A] [answered]
Hi,

Basically what I have written in the title. Although I can't see if it is explicitly mentioned in the integration tutorial, I assume the  processing steps (TFIDF, UMAP etc) is done on the "peaks" assay? And this is what is then used in the downstream integration (findintegrationanchors etc).

Thanks in advance!

**Comment by timoast** [comment, marked as answer]
Can you provide some more information? What is the difference between "peaks" and "ATAC" assays here?

**Reply by HaglundA** [reply]
Sorry - the "peaks" and "ATAC" assay are as per the 10X multiomic tutorial on the Signac website. 

So the "ATAC" assay is the resulting assay from the CreateChromatinAssay (using the peaks from the 10X h5 matrix and the fragment file) and the "peaks" assay is following the Peak Calling pipeline on the "ATAC" assay.

**Reply by timoast** [reply]
I think this is already addressed in the integration [vignette](https://satijalab.org/signac/articles/integrate_atac.html):

> An important first step in any integrative analysis of single-cell chromatin data is to ensure that the same features are measured in each dataset. Here, we quantify the multiome peaks in the ATAC dataset to ensure that there are common features across the two datasets.

**Reply by HaglundA** [reply]
Right, that makes sense. Thank you very much!
